pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -51,17 +51,17 @@ social:
 
 #top{style: "position:relative; /*height: #{imageheight}px; */ width:100%; overflow:hidden"}
 
-  #fullwidth0{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/fistbump.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 1; background-position: 70% 20%"}
+  #fullwidth0{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_0.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 1; background-position: 70% 20%"}
 
-  #fullwidth1{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/girl2.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
+  #fullwidth1{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_1.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
 
-  #fullwidth2{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url('/images/malala.jpg'); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 20%"}
+  #fullwidth2{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_2.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 20%"}
 
-  #fullwidth3{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/girl-learning.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
+  #fullwidth3{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_3.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
 
-  #fullwidth4{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/kids-learning.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 30% 40%"}
+  #fullwidth4{style: "z-index: -5; position:absolute; left: 0; top: 0; background-image: url(#{localized_image('/images/homepage_4.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 10%;"}
```

Comment:
Generally a space between the attribute and value, but if it's just 0 I think no space is fine. I'm going to fix this one that happens to be different from the rest and all the "top" attributes which have a space. Just for consistency.